plugins/woocommerce/src/Internal/Admin/ProductReviews/ReviewsListTable.php

```diff
@@ -1327,11 +1328,14 @@ protected function extra_tablenav( $which ) : void {
 	 */
 	protected function review_type_dropdown( $current_type ) : void {
 
-		$item_types = [
-			'all'     => __( 'All types', 'woocommerce' ),
-			'comment' => __( 'Replies', 'woocommerce' ),
-			'review'  => __( 'Reviews', 'woocommerce' ),
-		];
+		$item_types = apply_filters( 
+			'woocommerce_product_reviews_item_types',
+			[
```

Comment:
* As in an earlier comment, let's consider a more verbose hook name such as `woocommerce_product_reviews_list_table_item_types` to make it clear that the hook relates to the admin list table.
* Let's use the long array syntax (I realize short array syntax was already in use via the original code, but since we are touching it we should bring it into line with current standards).

Reply:
Adjusted naming of filter and added long array syntax